docs/source/tutorials/configurable.rst

```diff
@@ -0,0 +1,106 @@
+==============================
+Configuration and plugin setup
+==============================
+
+Maybe you :doc:`played with commands </tutorials/playing-command>` for your
+plugin and now you want to make your plugin configurable. If you run an
+instance of Sopel yourself, you probably had to open and edit its configuration
+file.
+
+Usually located in your home directory, under the ``.sopel/`` folder, the
+configuration file is an INI file with sections defined by Sopel's core and by
+plugins. In this tutorial, let's see how to declare and use a configuration
+section dedicated to your plugin.
+
+
+Declaring your configuration
+============================
+
+To declare a configuration section, you must first create a subclass of
+:class:`~sopel.config.types.StaticSection`, and define attributes::
+
+    from sopel.config import types
+
+    class MyFirstPlugin(types.StaticSection):
+        fruits = types.ListAttribute('fruits')
+
+
+Telling Sopel about it
+======================
+
+Now, having a class in your plugin doesn't achieve much: you need to tell the
+bot about it. The best place to do so is in the :func:`setup` function
+hook of your plugin::
+
+    def setup(bot):
+        bot.settings.define_section('firstplugin', MyFirstPlugin)
+
+This way, you tell Sopel that the section ``[firstplugin]`` is used by your
+plugin, and to parse this section Sopel must use your class, i.e.
+``MyFirstPlugin``.
+
+
+Using your section
+==================
+
+In your configuration file, you can add the following lines:
+
+.. code-block:: ini
+
+    [firstplugin]
+    fruits =
+        banana
+        apple
+        peach
+        strawberry
+
+And Sopel will expose that for you through ``bot.settings``. For example, you
+can write this command::
+
+    import random
+
+    @plugin.command('fruits')
+    def fruits(bot, trigger):
+        fruit = random.choice(bot.settings.firstplugin.fruits)
+        bot.say(f'I want a {fruit}!')
+
+And whenever someone triggers this command, the bot will say that it wants one
+of the configured fruits. If you want to list 50 fruits or only 2 is up to you,
+and to the bot owners who will install your plugin.
+
+
+Putting everything together
+===========================
+
+We can regroup everything together in one file::
```

Comment:
Further from the phrase shape used in the heading, so it doesn't sound as redundant.

```suggestion
We can combine all of this into one plugin file::
```

Yeah, nitpicks like this _are_ why I never get anything done at $work. Why do you ask?